twyla/service/message.py

```diff
@@ -17,32 +17,29 @@ def __init__(self, channel, body, envelope, name):
         self.envelope = envelope
         self.name = name
 
-    async def payload(self):
+    def payload(self):
         """
         The payload method is where the deserialization and validation of the
         event body happens. It returns an EventPayload object. The schema for
         deserialization and validation is loaded from a central schema service.
         """
-
-        try:
-            payload = EventPayload.from_json(self.body)
-        except ValidationError:
-            await self.drop()
-            raise
-
+        payload = EventPayload.from_json(self.body)
         return payload
```

Comment:
Why not just `return EventPayload.from_json(self.body)`?

Reply:
It used to be an async call, but you are right, the temporary variable is not needed anymore.